recipe/meta.yaml

```diff
@@ -98,6 +99,5 @@ about:
 extra:
   recipe-maintainers:
     - KrisThielemans
-    - olivierverdier
```

Comment:
Please don't remove maintainers without their explicit consent.

Reply:
@olivierverdier is no longer active in this field (unfortunately), since quite a few years. I don't mind keeping him in, but I presume he'd rather not get the spam. (He hasn't been able to react for a long time).

@olivierverdier just let me know